dev/tasks/dev/utils/git.js

```diff
@@ -97,6 +96,20 @@ module.exports = {
 	},
 
 	/**
+	 * Fetch all branches from each origin on selected repository.
+	 *
+	 * @param {String} repositoryLocation
+	 */
+	fetchAll( repositoryLocation ) {
```

Comment:
Please add a test for this function in `dev/tests/dev/git.js` 
![image](https://cloud.githubusercontent.com/assets/2396463/15218288/8a448ba0-185f-11e6-999a-2f676c812285.png)